app/Console/Commands/Permissions/SyncCommand.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+namespace App\Console\Commands\Permissions;
+
+use HMS\Entities\Role;
+use Illuminate\Console\Command;
+use App\Events\Roles\RoleCreated;
+use HMS\Repositories\RoleRepository;
+use Doctrine\ORM\EntityManagerInterface;
+use HMS\Repositories\PermissionRepository;
+use LaravelDoctrine\ACL\Permissions\Permission;
+
+class SyncCommand extends BaseCommand
+{
+    protected $permissionRepository;
+
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'permissions:sync';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Sync permissions and roles to match the default config';
+
+    /**
+     * @var array
+     */
+    private $permissions = [];
+
+    /**
+     * @var array
+     */
+    private $roles = [];
+
+    public function __construct(EntityManagerInterface $entityManager, RoleRepository $roleRepository, PermissionRepository $permissionRepository)
+    {
+        parent::__construct($entityManager, $roleRepository);
+
+        $permissions = config('roles.permissions');
+
+        foreach ($permissions as $permission) {
+            $this->permissions[$permission] = '';
+            $this->permissionRepository = $permissionRepository;
```

Comment:
Why is this in the loop?

Reply:
sublime must have auto inserted it there